migrations/0015-exhibition-publication-and-attributed-image-validation.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function(migration) {
+  const exhibitionPage = migration.editContentType('exhibitionPage');
+
+  exhibitionPage
+    .createField('datePublished')
+    .name('Publish at')
+    .type('Date')
+    .localized(false)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  const imageWithAttribution = migration.editContentType('imageWithAttribution');
+
+  imageWithAttribution
```

Comment:
This is awkward to scan for the changes applied as it has the full field definition. Just run the commands that result in actual changes to the field.